website/docs/faqs/dbt-specific-jinja.md

```diff
@@ -2,4 +2,4 @@
 title: What parts of Jinja are dbt-specific?
 ---
 
-There are certain expressions that are specific to dbt — these are documented in the [Jinja function reference](dbt-jinja-functions) section of these docs. Further, docs blocks, snapshots, and <Term id="materialization">materializations</Term> are custom Jinja _blocks_ that exist only in dbt.
+There are certain expressions that are specific to dbt — these are documented in the [Jinja function reference](dbt-jinja-functions/adapter) section of these docs. Further, docs blocks, snapshots, and <Term id="materialization">materializations</Term> are custom Jinja _blocks_ that exist only in dbt.
```

Comment:
```suggestion
There are certain expressions that are specific to dbt — these are documented in the [Jinja function reference](/reference/dbt-jinja-functions) section of these docs. Further, docs blocks, snapshots, and <Term id="materialization">materializations</Term> are custom Jinja _blocks_ that exist only in dbt.
```
@sophiad96 When I look at the deploy logs, I'm seeing that this is being resolved as a relative link and that's why it's broken (netlify thinks the link is under `FAQ/dbt-jinja-functions`). If you want to link to the dbt-jinja-functions main page, you can use the change I'm suggesting above. Is that a better link than the adapters sub page? 

<img width="992" alt="Screen Shot 2022-05-11 at 1 25 13 PM" src="https://user-images.githubusercontent.com/3880403/167941645-14470052-dc2e-4f2c-b519-5885b7646130.png">